src/analysis/modals/ComputeModal/AzureComputeModal/AzureComputeModal.test.ts

```diff
@@ -221,6 +230,7 @@ describe('AzureComputeModal', () => {
     const numberInput = await screen.getByLabelText('minutes of inactivity');
     expect(numberInput).toBeInTheDocument();
     await user.type(numberInput, '0');
```

Comment:
user.type has bad performance.  Switch these to just change the value of the input with fireEvent("change" ...)